lib/facets/aggs-all.js

```diff
@@ -21,7 +21,11 @@ module.exports = function (queryParams) {
         }
       },
       maker: {
-        filter: filter(queryParams),
+        filter: {
+          bool: {
+            must: filter(queryParams).bool.must.concat({ not: { terms: {'lifecycle.creation.maker.summary_title': ['Unknown maker', 'Unknown photographer']} } })
```

Comment:
I did that at first, filtered it out when creating the facets, but realised this should be quicker